views/endorsement-backers.js

```diff
@@ -0,0 +1,103 @@
+const { html } = require('../helpers')
+const stateNames = require('datasets-us-states-names-abbr')
+
+const legislatorTitles = {
+  California: ['Senator', 'Assembly'],
+  'Dane County': ['Supervisor'],
+}
+
+module.exports = (state, dispatch) => {
+  const { backersFilterQuery, measures, loading, location, votes } = state
+  const vote = votes[location.params.voteId]
+  const backers = vote.backers
+  const filteredBackers = backers && backers.reduce((memo, backer, index) => (
+    passesFilter({ ...backer, index: index + 1 }, backersFilterQuery)
+      ? memo.concat({ ...backer, index: index + 1 })
+      : memo
+  ), [])
+  const measure = measures[location.params.shortId]
+  const numLegislators = backers && backers.reduce((max, backer) => Math.max(max, backer.offices.length), 0)
+  const titles = legislatorTitles[measure.legislature_name] || []
+
+  return html`
+    <div>
+      ${loading.backers || !filteredBackers ? html`
+        <h3>Loading backers...</h3>
+      ` : html`
+        <div>
+          ${searchBar(dispatch, { backers, backersFilterQuery, filteredBackers })}
+          <table class="table is-narrow is-bordered is-striped" style="display: block; overflow-x: auto; max-width: 858px; border-right: 1px solid #e6e6e6">
+            <thead>
+              <tr>
+                <th></th>
+                <th>Time</th>
+                <th>Name</th>
+                <th>Location</th>
+                ${numLegislators > 0 ? html`<th>${titles[0] || 'Legislator'}</th>` : ''}
```

Comment:
Same here... not all legislatures have two representatives. Many city councils have a bunch of reps. Boise city council there would be six.

Reply:
Yeah. 
Was just trying to meet feature parity with the [existing 3 PoV documents](https://github.com/voteliquid/proof-of-vote/tree/e6d134fa41a839f35ef8159795a7cbb3f0aa889e/votes) for this initial version.

Reply:
I updated this to always show the correct number of reps: https://github.com/voteliquid/liquid.us/pull/155/commits/e2ab5510634242ca787ddc3799be788095f33d8e